public/apps/configuration/panels/auth-view/auth-view.tsx

```diff
@@ -23,17 +23,22 @@ import { AppDependencies } from '../../../types';
 import { ExternalLinkButton } from '../../utils/display-utils';
 import { getSecurityConfig } from '../../utils/auth-view-utils';
 import { InstructionView } from './instruction-view';
+import { DataSourceContext } from '../../app-router';
+import { SecurityPluginTopNavMenu } from '../../top-nav-menu';
 
 export function AuthView(props: AppDependencies) {
   const [authentication, setAuthentication] = React.useState([]);
   const [authorization, setAuthorization] = React.useState([]);
   const [loading, setLoading] = useState(false);
+  const { dataSource, setDataSource } = useContext(DataSourceContext)!;
 
   React.useEffect(() => {
     const fetchData = async () => {
       try {
         setLoading(true);
-        const config = await getSecurityConfig(props.coreStart.http);
+        const config = await getSecurityConfig(props.coreStart.http, undefined, {
```

Comment:
Yes - good call out. I will refactor this to be using named params, since a lot of places are not using body it's probably not a good idea to shim in this everywhere.

Reply:
Refactored in: [f46e83b](https://github.com/opensearch-project/security-dashboards-plugin/pull/1835/commits/f46e83b6d6e789d58cd8e04054a076afc5895392), but don't be afraid, most of it is just find and replace. This will be necessary as we transform more and more API calls to support multi datasource.